entrypoint.sh

```diff
@@ -49,5 +49,4 @@ git add . && \
 git commit -m "jekyll build from Action ${GITHUB_SHA}" && \
 git push --force $remote_repo master:$remote_branch && \
 rm -fr .git && \
-cd ..
-exit 0
+cd .. && exit 0 || exit 1
```

Comment:
could it be an alternative just to `exit $?`
I am not sure that it would work out of the box

Reply:
do you have the possibility to add a test on this ? :smile:

Reply:
In order to test manually, you can replace the `exit` with `echo $?` and make sure that the error code is propagated across all commands.

If this is the case, you can just remove the final exit command.

I assumed that the `set -e` would already make the script fail, probably you are missing some options (e.g. `-o pipefail`)

Reply:
Thanks for the suggestion, it works nicely